pleasew

```diff
@@ -74,12 +74,25 @@ get_profile () {
 # Check `PLZ_CONFIG_PROFILE` or fall back to arguments for a profile.
 PROFILE="${PLZ_CONFIG_PROFILE:-$(get_profile "${@}")}"
 
+# Find repo root by traversing up until we find .plzconfig
+find_repo_root() {
+    dir="$PWD"
+    while true; do
+        [ -f "$dir/.plzconfig" ] && echo "$dir" && return 0
+        [ "$dir" = "/" ] && return 1
```

Comment:
```suggestion
        [ "$dir" = "/" ] && return 0
```

As is, pleasew fails silently if run from outside a repository - because you can set a default repo in either `${HOME}/.config/please/plzconfig` or `/etc/please/plzconfig` (admittedly doing so in the latter would be a little bit mad), pleasew should continue even if it doesn't identify the repo root